h/static/styles/app.scss

```diff
@@ -50,6 +50,24 @@ body {
   }
 }
 
+.create-account-banner {
```

Comment:
On second thoughts, I think we can leave this as-is for the moment. As long as the component has its own class name, the markup and styling can be easily extracted later if necessary.